tests/finance/test_risk.py

```diff
@@ -317,6 +317,24 @@ def test_sharpe_value_when_null(self):
         )
         self.assertEqual(test_period['sharpe'], 0.0)
 
+    def test_sharpe_value_when_benchmark_null(self):
+        # Sharpe is displayed as '0.0' instead of np.nan
+        null_returns = factory.create_returns_from_list(
+            [0.0]*251,
```

Comment:
Why 251? Aren't there 252 trading days?

Reply:
I didn't investigate further, but seems like not on NYSE in 2006. This came from `trading_calendar.sessions_in_range(...)`.